package.json

```diff
@@ -38,7 +38,6 @@
     "dist"
   ],
   "peerDependencies": {
-    "fibers": ">= 3.1.0",
```

Comment:
Don't remove it from here, we still need to continue support (otherwise it will be breaking change), just improve readme about deprection

Reply:
sure. I will update readme and revert other changes

Reply:
done. Pls let me know any changes required